.gitignore

```diff
@@ -8,3 +8,4 @@ terraform.tfstate*
 /dist/
 state-migrate
 appgate/test-fixtures/appliance_customization_file_test.zip
+.idea/
```

Comment:
Save yourself some time, and add this to your local gitignore global https://gist.github.com/subfuzion/db7f57fff2fb6998a16c :)